runtime/v2/shim/util.go

```diff
@@ -33,6 +32,7 @@ import (
 	"github.com/gogo/protobuf/proto"
 	"github.com/gogo/protobuf/types"
 	"github.com/pkg/errors"
+	exec "golang.org/x/sys/execabs"
```

Comment:
I suggest also updating the comment down on line 75, since it's documenting the behaviour of `os/exec.LookPath` that you're replacing.

I'm not sure if anywhere else in the codebase does the extra 'check next to containerd executable' behaviour to find non-system-standard binaries, but they would have presumably similar explanatory comments that need updating..

Reply:
Good catch; rephrased the comment